libs/dml/src/default_value.rs

```diff
@@ -34,6 +38,11 @@ impl DefaultKind {
         matches!(self, DefaultKind::Expression(generator) if generator.name == "dbgenerated")
     }
 
+    /// Does this match @default(nanoid(_))?
+    pub fn is_nanoid(&self) -> bool {
+        matches!(self, DefaultKind::Expression(generator) if NANOID_RE.is_match(generator.name()))
```

Comment:
Do we really need a regex here, or could we just check `if generator.name().starts_with("nanoid(")`?

Reply:
done! the regex is gone ❤️